ClearKeep/Scenes/PeerCall/Views/SingleVoiceCallInfoView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SingleVoiceCallInfoView.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 25/07/2022.
+//
+
+import SwiftUI
+import CommonUI
+
+struct SingleVoiceCallInfoView: View {
+	@ObservedObject var viewModel: CallViewModel
+	
+	func image(withName: String?) -> Image? {
+		guard let name = withName, !name.isEmpty else {
+			return nil
+		}
+		return Image(name)
+	}
+	
+	var body: some View {
+		GeometryReader { reader in
+			VStack {
+				Spacer()
+				
+				VStack(spacing: 32) {
+					Spacer()
+						.frame(height: 60)
+					
+					if viewModel.callType == .audio, viewModel.callStatus != .answered, !viewModel.getStatusMessage().isEmpty {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/PeerCall/Views/SingleVoiceCallInfoView.swift#L30"></span>Arguments can be omitted when matching enums with associated values if they are not used.<br />
<code>empty_enum_arguments</code> <code>SingleVoiceCallInfoView.swift:30</code></td>
    </tr>
  </tbody>
</table>